Docs/Configuration.tex

```diff
@@ -5511,7 +5513,7 @@ \subsection{SMBIOS Properties}\label{platforminfosmbios}
 \item
   \texttt{FirmwareFeaturesMask}\\
   \textbf{Type}: \texttt{plist\ data}, 8 bytes\\
-  \textbf{Failsafe}: \texttt{0}\\
+  \textbf{Failsafe}: \texttt{0} (OEM specified on Apple hardware, 0 otherwise)\\
```

Comment:
Same thing once more.

Reply:
Ditto. Thanks for this feedback.

Reply:
To be honest, I am not sure whether there is any special hack for these parts, but in general the Failsafe value can be found in [OcConfigurationLib.h](https://github.com/acidanthera/OpenCorePkg/blob/master/Include/Acidanthera/Library/OcConfigurationLib.h).

Reply:
Oh, I see - again I'm not at my Mac now, but here I was trying to follow the convention for e.g. some of the newly added (by @vit9696!) behaviour in the generic section, where it now says what the failsafe value actually is, first (e.g. empty string) and then say what it means, afterwards, in brackets (e.g. "OEM specified or not installed", which means in very abbreviated form that either result can occur, depending on details (in those cases, of which section(s) it is being applied to and whether a value exists in each section)). The details of how these features, which I'm adding, are intended to work are similar (I believe!).

Reply:
Alright. Apologies too. I guess I am ok to live with the new style of failsafe. :)

Reply:
Tbh I think there might be a fairly strong case for _not_ moving the failsafe value's _meaning_ up into brackets next to the value itself - because as I just mentioned it results in *very* abbreviated descriptions.

For instance spelling out in more detail, not less, what I just said about the meaning of the OEM behaviour in the Generic section would be very helpful, I think, to the average user coming to the docs for the first time.

NB This point has a direct bearing on the ongoing review by @vit9696 and @dakanji of @dakanji's new proposed changes. (And it may well make more sense to address this separately - or not - after that round of changes.)

But yes, I think it makes most sense for me to stick with the current new style for now, as you've just suggested.